PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -1017,6 +1017,13 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	//  	...
 	//  }
 
+	#define NPPM_GETTOOLBARICONSETMODE (NPPMSG + 118)
+	// BOOL NPPM_GETTOOLBARICONSETMODE(0, 0)
+	// Get Notepad++ Toobar Icon Set mode (ENUM Range: 0 to 4).
+	// wParam: 0 (not used)
+	// lParam: 0 (not used)
+	// Return Toolbar Icon Set mode as an integer value [ENUM range: 0 (TB_SMALL - Fluent UI: small) to 4 (TB_STANDARD - Standard icons: small)].

```

Comment:
Please use the following comment instead:
// Return toolbar icon set choice as an integer value. Here are 5 possible values: 
// 0 (Fluent UI: small), 1 (Fluent UI: large), 2 (Filled Fluent UI: small), 3 (Filled Fluent UI: large) and 4 (Standard icons: small).

Reply:
Implemented in the new commit: "Renamed message to NPPM_GETTOOLBARICONSETCHOICE"